tests/browser_test.go

```diff
@@ -141,14 +142,21 @@ func TestBrowserUserAgent(t *testing.T) {
 }
 
 func TestBrowserCrashErr(t *testing.T) {
```

Comment:
Do we still need `t.Parallel()`?

Reply:
`t.Parallel()` can not work with `t.Setenv()` :(
To be honest I was considering removing that test completely, as I'm not sure it provides much value and now its testing has become more complex as we have to capture the error from the Goja layer..

Reply:
Aha, i see 👍